src/Leaderboard.js

```diff
@@ -0,0 +1,23 @@
+export default class Leaderboard {
+  #BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+  #GAME_ID = '6RWYo73TvPoVLbLnw9Tv';
+
+  async getResults() {
```

Comment:
- [ ] As mentioned above Let's use `arrow functions` instead of the standard functions.